bin/run_ckan.sh

```diff
@@ -84,5 +86,12 @@ docker run --name ckan \
 docker exec ckan /usr/local/bin/ckan-paster --plugin=ckan datastore set-permissions -c /etc/ckan/production.ini | \
   docker exec -i ckan-datastore-db psql -U ckan
 
-docker exec ckan /usr/local/bin/ckan-paster --plugin=ckanext-collaborators collaborators init-db -c /etc/ckan/production.ini
+#docker exec ckan /usr/local/bin/ckan-paster --plugin=ckanext-collaborators collaborators init-db -c /etc/ckan/production.ini
+
+# Create Sysadmin
+docker exec -it ckan /usr/local/bin/ckan-paster --plugin=ckan sysadmin -c /etc/ckan/production.ini add johndoe
+
+# Don't forget to:
 
+# sudo certbot certonly --standalone -d <FQDN e.g. data.demo.com>
+docker run -d --restart always -v /etc/letsencrypt:/etc/nginx/certs:z -v /home/ubuntu/Data-Portal/config/default.conf:/etc/nginx/conf.d/default.conf --network ckan --name ckan-proxy -p 443:443 -p 80:80 nginx
```

Comment:
I'm not keen for the gateway proxy to live in the CKAN setup script, mostly for security reasons.

Ideally, we would spin up ckan, run some sort of local readiness test, then run proxy setup script. Will you please move proxy setup into its own script, please